infrastructure/main.tf

```diff
@@ -1,16 +1,35 @@
+locals {
```

Comment:
Nicely done, can easily create resources with good and consistent naming now :)

Reply:
Thank you dude! :)